tests/roaring64_unit.cpp

```diff
@@ -1776,6 +1776,27 @@ DEFINE_TEST(test_iterator_read) {
     roaring64_bitmap_free(r);
 }
 
+DEFINE_TEST(test_stats) {
+    // create a new empty bitmap
+    roaring64_bitmap_t* r1 = roaring64_bitmap_create();
+    assert_non_null(r1);
+    // then we can add values
+    for (uint32_t i = 100; i < 1000; i++) {
+        roaring64_bitmap_add(r1, i);
+    }
+    for (uint32_t i = 1000; i < 100000; i += 10) {
+        roaring64_bitmap_add(r1, i);
+    }
+    roaring64_bitmap_add(r1, 100000);
+
+    roaring64_statistics_t stats;
+    roaring64_bitmap_statistics(r1, &stats);
+    assert_true(stats.cardinality == roaring64_bitmap_get_cardinality(r1));
+    assert_true(stats.min_value == 100);
+    assert_true(stats.max_value == 100000);
```

Comment:
Should use `assert_int_equal` here, we'll get a much better error if the assert ever fails.